staging.gen3.biodatacatalyst.nhlbi.nih.gov/portal/gitops.json

```diff
@@ -1,5 +1,9 @@
 {
   "gaTrackingId": "UA-119127212-5",
+  "grafanaFaroConfig": {
+    "grafanaFaroEnable": true,
+    "grafanaFaroNamespace": "bdcatprod"
```

Comment:
This needs to go under gen3.biodatacatalyst.nhlbi.nih.gov/portal/gitops.json

Reply:
fixed